wp-admin/js/widgets/media-image-widget.js

```diff
@@ -98,7 +98,7 @@
 
 			if ( ! _.isEmpty( attachment ) ) {
 				props = {
-					attachment_id: attachment.id,
+					attachment_id: 0,
```

Comment:
This causes my control preview to not update some times. But not always. Still trying to work out why. Does this work for everyone else?